rust/tvm-rt/src/array.rs

```diff
@@ -118,6 +125,12 @@ impl<T: IsObjectRef> IntoIterator for Array<T> {
     }
 }
 
+impl<T: IsObjectRef> FromIterator<T> for Array<T> {
+    fn from_iter<I: IntoIterator<Item = T>>(iter: I) -> Self {
+        Array::from_vec(iter.into_iter().collect()).unwrap()
```

Comment:
Inconsistency: `FromIterator::from_iter()` is implemented elsewhere using
```
Array::from_vec(iter)
```
See the following similar code:
* array.rs:92
* graph_mod.rs:139
* graph_mod.rs:151

For more information about this comment, see https://fixie.ai/comment/55732b1c0.